### ISSUE TITLE
ERROR on promoting app with volumes

### ISSUE THREAD

**Issue by RyanBertrand** [convox/convox#692]
Rack Version: 3.13.5

Error:
`Promoting RGUFGAVKBJW... ERROR: template: :10:22: executing "" at <.Labels>: can't evaluate field Labels in type string
`

If I remove the "volumes" entry, it deploys fine. This seems to be a new issue as we recently updated our Rack version and can no longer deploy this app.  

convox.yaml
```
services:
  api:
    build: .
    internal: true
    singleton: true
    health:
      path: /health
      grace: 30
    port: 8108
    scale:
      count: 1
      memory: 4096
      cpu: 1024
    environment:
      - TYPESENSE_API_KEY
    volumes:
      - /var/typesense/data
```

**Comment by RyanBertrand** [comment]
Resolved with 3.13.6rc1

**Comment by nightfury1204** [comment]
> Resolved with 3.13.6rc1

@RyanBertrand please use 3.13.6 instead
